geode-core/src/main/java/org/apache/geode/cache/RegionFactory.java

```diff
@@ -100,6 +100,16 @@ protected RegionFactory(InternalCache cache, RegionAttributes ra) {
     this.attrsFactory = new AttributesFactory<K, V>(ra);
   }
 
+  /**
+   * For internal use only.
+   *
+   * @since GemFire 9.10
```

Comment:
The since should be a geode release not a gemfire release

Reply:
I modified that in the latest change set.